.github/config/android-x86.json

```diff
@@ -19,6 +19,7 @@
         "system_webview_shell_apk",
         "cobalt:gn_all"
     ],
+    "test_on_device": true,
```

Comment:
Didn't we turn off testing on x86? Do we want to re-enable it?

Reply:
removed.